src/users.rs

```diff
@@ -2,6 +2,7 @@
 
 pub mod create_or_modify;
 pub mod get_details;
+pub mod list;
```

Comment:
```suggestion
pub mod list_users;
```

https://github.com/ruma/ruma/blob/main/CONTRIBUTING.md#naming-endpoints